tests/test_load_dump.py

```diff
@@ -594,3 +594,68 @@ def test_json_infinities_get_stringified(self):
         assert data["migrations"][0]["start_time"] == "Infinity"
         g2 = demes.loads(json_str, format="json")
         g2.assert_close(g1)
+
+
+class TestMultiDocument:
```

Comment:
Can you check the empty file case, where we should return an empty genereator?

Reply:
Ah, good call! An empty genmerator sounds like the right thing to do.